### PR TITLE
fix examples and update to 2023 R2

### PR DESCRIPTION
**Pull request by maxcapodi78**

### CODE REVIEW

**Review by gmalinve — APPROVED** [review]

**Review by MaxJPRey — APPROVED** [review]
LGTM.

File: examples/07-EMIT/EMIT_HFSS_Example.py [hunk 1]
```diff
@@ -111,7 +111,7 @@
 #
 # This part of the example requires Ansys AEDT 2023 R2. 
 
-if desktop_version > "2023.1":
+if desktop_version > "2023.2":
```
Comment (jsalant22) [inline examples/07-EMIT/EMIT_HFSS_Example.py:114]:
```suggestion
if desktop_version > "2023.1":
```

File: examples/07-EMIT/EMIT_Example.py [hunk 2]
```diff
@@ -66,7 +66,7 @@
 #
 # This part of the example requires Ansys AEDT 2023 R2. 
 
-if desktop_version > "2023.1":
+if desktop_version > "2023.2":
```
Comment (jsalant22) [inline examples/07-EMIT/EMIT_Example.py:69]:
```suggestion
if desktop_version > "2023.1":
```

File: examples/07-EMIT/ComputeProtectionLevels.py [hunk 3]
```diff
@@ -55,7 +55,7 @@ def install(package):
 # Launch AEDT with EMIT. The ``Desktop`` class initializes AEDT and starts it
 # on the specified version and in the specified graphical mode.
 
-if desktop_version <= "2023.1":
+if desktop_version <= "2023.2":
```
Comment (jsalant22) [inline examples/07-EMIT/ComputeProtectionLevels.py:58]:
```suggestion
if desktop_version <= "2023.1":
```

File: examples/07-EMIT/ComputeInterferenceType.py [hunk 4]
```diff
@@ -40,7 +40,7 @@ def install(package):
 
 # Check for if emit version is compatible
 desktop_version = "2023.2"
-if desktop_version <= "2023.1":
+if desktop_version <= "2023.2":
```
Comment (jsalant22) [inline examples/07-EMIT/ComputeInterferenceType.py:43]:
```suggestion
if desktop_version <= "2023.1":
```

File: examples/07-EMIT/ComputeChannelSeparation.py [hunk 5]
```diff
@@ -61,7 +61,7 @@ def install(package):
 # Launch AEDT with EMIT. The ``Desktop`` class initializes AEDT and starts it
 # on the specified version and in the specified graphical mode.
 
-if desktop_version <= "2023.1":
+if desktop_version <= "2023.2":
```
Comment (jsalant22) [inline examples/07-EMIT/ComputeChannelSeparation.py:64]:
```suggestion
if desktop_version <= "2023.1":
```

File: examples/07-EMIT/EMIT_HFSS_Example.py (hunk 1 above)
Comment (maxcapodi78) [reply examples/07-EMIT/EMIT_HFSS_Example.py]: Fixed. Sorry for the mistake

**Comment by maxcapodi78** [comment]
@jsalant22  I had to comment 1 example since continue to fails on my local machine and on build machine

**Pull request merged**